README.md

```diff
@@ -6,7 +6,4 @@ A Python 3.6 implementation of the [Request Network API](https://github.com/Requ
 To contribute please see the Wiki!
 
 # Dependencies
-ipfsapi (pip install ipfsapi)
-asyncio (pip install asyncio)
-
-
+Install dependencies by using pip install -r requirements.txt
```

Comment:
Surround text with ` (backtick) to format it as code or a command (in this case around the pip command)